.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,30 +45,14 @@ body:
     validations:
       required: false
 
-  - type: input
-    id: platform-version-if-selfhosting
-    attributes:
-      label: If you're self-hosting Renovate, tell us what version of the platform you run.
-      description: |
-        This field is for the version number of your _platform_, so one of these:
-          - AWS CodeCommit
-          - Azure DevOps
-          - Bitbucket Cloud
-          - Bitbucket Server
-          - Azure DevOps
-          - Gitea or Forgejo
-          - GitHub (.com and Enterprise Server)
-          - GitLab (.com and CE/EE)
-    validations:
-      required: false
-
   - type: dropdown
     id: regression-error
     attributes:
       label: Was this something which used to work for you, and then stopped?
       options:
+        - '-- Please select from the below --'
```

Comment:
I'm seeing this error message when I view this file on GitHub:

> There is a problem with this template
>
> YAML syntax error: (): did not find expected '-' indicator while parsing a block collection at line 53 column 9. [Learn more about this error.](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms)

You probably need to drop the `--` characters to get the form to work properly.

Link to the warning: https://github.com/renovatebot/renovate/blob/8bde1049b47afb6ebbee410105e909f3bf2fa5cf/.github/ISSUE_TEMPLATE/bug_report.yml

Reply:
```suggestion
        - 'Placeholder value, please select the correct response from the dropdown'
```